packages/esm-patient-registration-app/src/patient-registration/validation/patient-registration-validation.test.tsx

```diff
@@ -112,18 +210,19 @@ describe('Name input', () => {
     fireEvent.blur(middleNameInput);
     fireEvent.change(familyNameInput, { target: { value: familyNameValue } });
     fireEvent.blur(familyNameInput);
+    screen.debug();
 
     return {
-      givenNameError: screen.queryByText('Given name is required'),
-      middleNameError: screen.queryByText('Middle name is required'),
-      familyNameError: screen.queryByText('Family name is required'),
+      givenNameError: screen.queryByText('First Name is required'),
+      middleNameError: screen.queryByText('Middle Name is required'),
+      familyNameError: screen.queryByText('Family Name is required'),
     };
   };
 
   testValidName('Aaron', 'A', 'Aaronson');
-  testValidName('No', '', 'Middle Name');
-  testInvalidName('', '', '', 'Given name is required', 'givenNameError');
-  testInvalidName('', '', '', 'Family name is required', 'familyNameError');
-  testInvalidName('', 'No', 'Given Name', 'Given name is required', 'givenNameError');
-  testInvalidName('No', 'Family Name', '', 'Family name is required', 'familyNameError');
+  // testValidName('No', '', 'Middle Name');   // this test fails because the error message is showing up for the middle name
```

Comment:
What kind of error message does it show?

Reply:
Middle Name is required is the error message but it shouldn't show because this field is optional.

Reply:
@Piumal1999 I fixed this test. Thank you. Can you please first review the mocks for `input` and `formik` components. I have doubt on them.